tests/docker_adapter_test/swarm_docker_volume.tf

```diff
@@ -0,0 +1 @@
+../../mcp/swarm_docker_volume.tf
```

Comment:
Which set of tests are we using? It feels like these are all duplicates?

Reply:
tests/docker_adapter_test - used in local testing